### DISCUSSION TITLE
Exclude plugins defined in configuration

### DISCUSSION THREAD

**Discussion by edwincoronado** [swc-project/swc#2788] [Q&A]
Hello,

Babel has the ability to [exclude plugins](https://babeljs.io/docs/en/babel-preset-env#exclude) like regenerator runtime in its configuration such as:

```js
{
   "presets": [
      ["env", {
         "exclude": ["transform-regenerator"]
      }]
   ]
}
```

Is anybody aware on how to accomplish this in swc? I can't seem to find it in the docs. 

Thanks!

**Comment by kwonoj** [comment]
Probably not the answer you're looking for, but in my case I'm setting target higher and specifically include necessary transforms instead.
